genaisrc/pr.advisor.genai.mts

```diff
@@ -0,0 +1,15 @@
+script({
+    title: "PR Title Evaluator",
+    description: "A tool that evaluates pull request titles against predefined code standards.",
+    model: "github:gpt-4o"
+})
+
+const csvs = await workspace.findFiles("**/*.csv")
+def("REPORTS", csvs)
+
+const instructions = await workspace.readText(".github/pr-title-guidelines.md")
+def("CODE_STANDARDS", instructions.content)
+
+$`## Task
+Evaluate titles in the REPORTS against the CODE_STANDARDS and give a brief analysis of the results.
```

Comment:
To trigger warnings/errors in vscode format:

`Report errors or warnings using annotations.`

Reply:
![image](https://github.com/user-attachments/assets/decfe918-bbc1-44c9-84c1-9fb39486817d)